### PR TITLE
Make pdf fields searchable. Refs #2301

### PR DESCRIPTION
**Pull request by pilz**

### CODE REVIEW

File: src/recensio/plone/behaviors/base.py [hunk 1]
```diff
@@ -83,6 +83,7 @@ class IBase(model.Schema):
         title=_("Generated Pdf"),
         required=False,
     )
+    searchable("generatedPdf")
```
Comment (reinhardt) [inline src/recensio/plone/behaviors/base.py:86]:
```suggestion
```

This is generated from the field `review` which is already searchable.

**Review by reinhardt — CHANGES_REQUESTED** [review]
Please run isort to fix the checks.

File: src/recensio/plone/behaviors/base.py (hunk 1 above)
Comment (pilz) [reply src/recensio/plone/behaviors/base.py:86]: Agreed

File: src/recensio/plone/behaviors/base_review.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 from plone.autoform import directives
 from plone.autoform.interfaces import IFormFieldProvider
+from plone.app.dexterity.textindexer import searchable
```
Comment (reinhardt) [inline src/recensio/plone/behaviors/base_review.py:3]:
```suggestion
from plone.app.dexterity.textindexer import searchable
from plone.autoform import directives
from plone.autoform.interfaces import IFormFieldProvider
```

**Review by reinhardt — APPROVED** [review]

**Pull request merged**